i18n/default/src/AssetsPage/index.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "id": "AssetsPage.title",
+    "description": "Title for AssetsPage",
+    "defaultMessage": "Files & Uploads"
+  }
+]
```

Comment:
Note that the extracted messages are [MessageDescriptor objects](https://github.com/yahoo/react-intl/wiki/Components#message-descriptor)

Reply:
Is this thinking that this is the file that we feed up to transifex for translation? Something to think about is that we'll eventually have more pages than just this one, and ideally we wouldn't have to specify to transifex each new page to translate explicitly.

Reply:
Yeah, the thinking here is that we can use "every json file under `i18n/default` " as our input to transifex. I've currently got babel configured to extract all relevant display strings under `src/` into `i18n/default/src/`, mirroring the directory structure.

If the tooling is set up correctly, we shouldn't ever need devs to touch files under `i18n/` explicitly - babel tools will extract inline strings from `src/` and place them under `i18n/default/src`, where we can easily feed them to transifex. Then transifex can output the json under `i18n/<locale>`, for all locales.

This works well for transifex, but as it's currently set up we're placing *all* translations for a given locale into a single file - should we consider keeping the transifex output stored per-page, to optimize bundle sizing?